cmd/kubelet/app/server.go

```diff
@@ -409,7 +409,7 @@ func UnsecuredDependencies(s *options.KubeletServer, featureGate featuregate.Fea
 // not be generated.
 func Run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Dependencies, featureGate featuregate.FeatureGate) error {
 	// To help debugging, immediately log version
-	klog.InfoS("Kubelet version", "kubeletVersion", version.Get())
+	klog.InfoS("kubelet version", "kubeletVersion", version.Get())
```

Comment:
This contradicts with the accepted logging style I guess. Log messages should be capitalized.